lib/api/routes/userRoute.js

```diff
@@ -53,7 +51,7 @@ userRouter.post('/', async (req, res) => {
     res.send(new Error('Passwords does not match'));
     return;
   }
-  if (nullOrEmpty(username) || nullOrEmpty(password) || nullOrEmpty(password2)) {
+  if (!username || !password || !password2) {
```

Comment:
See the previous pr. your check is not the same as mine.
```
let a = 0;
nullOrEmpty(a); //false
!a; //true
```

Reply:
in my case I want to explicitly check for null or empty

Reply:
Sounds good, I changed it back to the same logic but slimer:

```js
const nullOrEmpty = (str) => str == null || str.length === 0;
```

is the same as
```js
const nullOrEmpty = (str) => Boolean(str?.length);
```

as the `if`-block will convert everything inside it's brackets to a boolean value for checking, we can inline it:

```js
  if (!username?.length || !password?.length || !password2?.length) {
```

Let me know if that works for you.